app/dashboard/models.py

```diff
@@ -2076,6 +2079,27 @@ def tips(self):
         tipped_for = Tip.objects.filter(username__iexact=self.handle).order_by('-id')
         return on_repo | tipped_for
 
+    def calculate_and_save_persona(self):
+        network = settings.ENABLE_NOTIFICATIONS_ON_NETWORK
+        designation = None
+        with connection.cursor() as cursor:
+            try:
+                cursor.execute(PERSONA_SQL, [network, self.id, network])
+                result = cursor.fetchone()
+                _, _, designation, _, _ = result
+            except Exception as e:
+                logger.info(
+                    'Exception calculating persona for user %s: %s' % (self.id,
```

Comment:
should we keep this logger info in one line ?

Reply:
I know it looks a bit weird, but its pep8 style: https://www.python.org/dev/peps/pep-0008/